metahtml/timestamp.py

```diff
@@ -361,24 +380,45 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'militarytimes.com',              '//meta[@itemprop="datePublished"]/@content'),
         ( 'mundiario.com',                  '//span[@class="content-time"]' ),
         ( 'nbc12.com',                      '//meta[@name="date"]/@content'),
+        ( 'newsbreak.com',                  '//meta[@itemprop="dateCreated"]/@content' ),
+        ( 'newsbreak.com',                  '//meta[@property="article:published_time"]/@content '),
+        ( 'news.un.org',                    '//meta[@property="article:published_time"]/@content' ),
+        ( 'ngong.net',                      '//meta[@property="published_time"]/@content' ),
         ( 'nydailynews.com',                '//meta[@name="date"]/@content'),
         ( 'nytimes.com',                    '//meta[@property="article:published"]/@content' ),
+        #( 'nytimes.com',                    '//meta[@name="pdate"]/@content' ),
```

Comment:
We typically don't want commented code to ever enter the git repo.  Only actual substantive lines should be added.